src/rard/utils/shared_functions.py

```diff
@@ -40,3 +40,46 @@ def reassign_to_unknown(worklink):
     worklink.definite_work = False
     worklink.definite_book = False
     worklink.save()
+
+
+def collate_unknown(instance):
+    """This makes sure there's only one unknown work/book per antiquarian/work and combines contents if otherwise"""
+    if instance.__class__.__name__ == "Antiquarian":
```

Comment:
iffing on class name? Strongly consider adding a `collate_unknown` method to Antiquarian and Work model objects instead.